Doc/Suggested_features.md

```diff
@@ -1,22 +1,22 @@
-Suggestions for upgrade
+# Enhancements and Feature Requests
```

Comment:
Much of the change set from here-on, below, is simply cleaning up md-files format, to make the docs a bit more readable & useful.